lib/rubocop/cop/rspec/stubbed_mock.rb

```diff
@@ -106,10 +106,13 @@ class StubbedMock < Base
         #   @param node [RuboCop::AST::Node]
         #   @yield [RuboCop::AST::Node] matcher
         def_node_matcher :matcher_with_hash, <<~PATTERN
-          {
-            (send nil? :receive_messages hash)           # receive_messages(foo: 'bar', baz: 'qux')
-            (send nil? :receive_message_chain ... hash)  # receive_message_chain(:foo, bar: 'baz')
-          }
+          (send nil?
+            {
+              :receive_messages hash           # receive_messages(foo: 'bar', baz: 'qux')
+              |
+              :receive_message_chain ... hash  # receive_message_chain(:foo, bar: 'baz')
```

Comment:
wonder if it won't be fixed by just having `<hash ...>`

Reply:
Not sure what you mean exactly. `receive_messages`/`receive_message_chain` differ here, and the latter can have positional arguments, while the former can't.
Reverting this as well.